src/utils/canAccessLocalStorage.ts

```diff
@@ -0,0 +1,8 @@
+export const canAccessLocalStorage = () => {
+  try {
+    localStorage.getItem('test');
+    return true;
+  } catch {
+    return false;
+  }
+};
```

Comment:
How would you feel about co-locating this in the `LayoutManager` directory as that's the only place using it at the moment?

Reply:
sure!